### PR TITLE
DOC: Add new and retired Council members

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Review by jjomier — APPROVED** [review]
LGTM - Thanks for having me back!

File: src/pages/council.js [hunk 1]
```diff
@@ -41,30 +41,24 @@ const CouncilPage = () => {
...
-            Jon Haitz Legarreta Gorroño - PhD Student, Université de Sherbrooke
-          </li>
-          <li>
-            Lauren O'Donnell - Brigham and Women's Hospital - Harvard Medical
-            School
-          </li>
+          <li>Jon Haitz Legarreta Gorroño - Postdoctoral Research Fellow, Brigham and Women&#39;s Hospital, Harvard Medical School</li>
           <li>Ipek Oguz - University of Pennsylvania </li>
-          <li>Beatriz Paniagua - Kitware</li>
+          <li>Dženan ZukiĆ - Kitware</li>
```
Comment (jhlegarreta) [inline src/pages/council.js:51]: ZukiĆ : Zukić ?

File: src/pages/council.js [hunk 2]
```diff
@@ -41,30 +41,24 @@ const CouncilPage = () => {
...
+          <li>Julien Jomier - NVIDIA</li>
+          <li>Serena Bonaretti - Balgrist Campus</li>
         </ul>
-        <p>Founding Members include:</p>
+        <p>Past Council Members include:</p>
         <ul>
-          <li> Elsa Angelini - Columbia</li>
+          <li>Paul Yushkevich - UPenn</li>
+          <li>Lauren O'Donnell - Brigham and Women&#39;s Hospital - Harvard Medical School</li>
```
Comment (jhlegarreta) [inline src/pages/council.js:58]: `Women&#39;s` : Women's ?

File: src/pages/council.js [hunk 3]
```diff
@@ -41,30 +41,24 @@ const CouncilPage = () => {
...
+          <li>Beatriz Paniagua - Kitware</li>
+
```
Comment (jhlegarreta) [inline src/pages/council.js:60]: Remove blank line ?

File: src/pages/council.js (hunk 2 above)
Comment (thewtex) [reply src/pages/council.js:58]: The escaped version is intentionally for HTML.

File: src/pages/council.js (hunk 3 above)
Comment (thewtex) [reply src/pages/council.js:60]: It is there to mark different elections.

File: src/pages/council.js [hunk 4]
```diff
@@ -41,30 +41,24 @@ const CouncilPage = () => {
...
         <ul>
           <li>Andras Lasso - Queen's University</li>
-          <li>
-            Jon Haitz Legarreta Gorroño - PhD Student, Université de Sherbrooke
-          </li>
-          <li>
-            Lauren O'Donnell - Brigham and Women's Hospital - Harvard Medical
-            School
-          </li>
+          <li>Jon Haitz Legarreta Gorroño - Postdoctoral Research Fellow, Brigham and Women&#39;s Hospital, Harvard Medical School</li>
```
[the remark this answers is not in the document]
Comment (thewtex) [reply src/pages/council.js:49]: It is there intentionally for HTML.

**Review by jhlegarreta — APPROVED** [review]

**Pull request merged**